x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch/metric_set_index_detail.js

```diff
@@ -75,6 +75,10 @@ export const metricSet = {
     {
       keys: ['index_segment_count_total', 'index_segment_count_primaries'],
       name: 'index_segment_count'
+    },
+    {
+      keys: ['index_index_latency', 'index_query_latency'],
+      name: 'index_latency'
```

Comment:
@chrisronline It's here that have have 2 metrics combined in 1 graph. Where do I modify the title of the graph. I tried to create an "empty" metric `index_latency` but this trick didn't work. It still picks the name of the first metric.

Reply:
Right. It uses the title for the first metric found in the set, so in this case `index_index_latency`. That's what you're seeing right? That's the behavior I found as well

Reply:
Yes, but in this case the title should not be from the first metric. And there are quite a few examples in Advance where this is also not the case but I didn't find out so far how this is changed. @pickypg might share some insights here?